.devcontainer/scripts/.zshrc

```diff
@@ -1,3 +0,0 @@
-#!/bin/zsh
```

Comment:
The two .zshrc files should stay, and be copied, so that developers using the devcontainer get hermit accessible on their path.